rero_ils/modules/items/listener.py

```diff
@@ -38,3 +38,4 @@ def enrich_item_data(sender, json=None, record=None, index=None,
         json['organisation'] = {
             'pid': org_pid
         }
+        json['available'] = item.available
```

Comment:
is it really required as it has been added to the dump?

Reply:
Yes, we need the field at the level of record and list of records.